File: C#/Algorithms/Search/LinearSearch.cs
/**
 * Class LinearSearch
 */
public class LinearSearch {

    /**
     * Implements Linear Search algorithm
     *
     * @param array array
     * @param int targetElement
     * @returns int
     */
    public int linearSearch(int[] array, int targetElement) {
        int arrayLength = array.Length;

        for (int i = 0;i < arrayLength;i++) {
            if (array[i] == targetElement) {
                return i;
            }
        }

        return -1;
    }
}

File: PHP/Algorithms/Search/linearSearch.php
<?php

/**
 * Implements Linear Search algorithm
 *
 * @param array $array
 * @param mixed $searchingElement
 * @return int
 */
function linearSearch(array $array, $searchingElement)
{
    foreach ($array as $key => $value) {
        if ($value == $searchingElement) {
            return $key;
        }
    }

    return -1;
}
